default-templates/new-account/index.html

```diff
@@ -14,10 +14,10 @@
   <div class="row">
     <div class="col-md-12">
       <p style="margin-top: 3em; margin-bottom: 3em;">
-        Welcome to your Solid user profile.
+        Welcome.
       </p>
       <p>
-        Your Web ID is:<br />
+        Web ID:<br />
```

Comment:
Would it make sense to make this more descriptive? As in `The owner of this page/space is {{webId}}`? There was some confusion in the past with "Your Web ID is", and this removes some confusion, but maybe not all.